native/membrane_videocompositor_wgpu/src/compositor.rs

```diff
@@ -19,60 +69,19 @@ impl Vertex {
     };
 }
 
-const UPPER_VERTICES: [Vertex; 4] = [
-    Vertex {
-        position: [1.0, 1.0, 0.0],
-        texture_coords: [1.0, 0.0],
-    },
-    Vertex {
-        position: [-1.0, 1.0, 0.0],
-        texture_coords: [0.0, 0.0],
-    },
-    Vertex {
-        position: [-1.0, 0.0, 0.0],
-        texture_coords: [0.0, 1.0],
-    },
-    Vertex {
-        position: [1.0, 0.0, 0.0],
-        texture_coords: [1.0, 1.0],
-    },
-];
-
-const LOWER_VERTICES: [Vertex; 4] = [
-    Vertex {
-        position: [1.0, 0.0, 0.0],
-        texture_coords: [1.0, 0.0],
-    },
-    Vertex {
-        position: [-1.0, 0.0, 0.0],
-        texture_coords: [0.0, 0.0],
-    },
-    Vertex {
-        position: [-1.0, -1.0, 0.0],
-        texture_coords: [0.0, 1.0],
-    },
-    Vertex {
-        position: [1.0, -1.0, 0.0],
-        texture_coords: [1.0, 1.0],
-    },
-];
-
 pub struct State {
```

Comment:
I think we should keep something like `output_caps` as `State` attribute or at least scene `width` and `height` (it's useful for some transformations to have information about scene resolution). Maybe we could have two structs `State` and `Scene` and split those responsibilities somehow? I think it's one of the topics to discuss at the next architecture design meeting.